core/src/main/java/org/bitcoinj/crypto/KeyType.java

```diff
@@ -20,7 +20,7 @@ public enum KeyType {
     RSA(0),
     ECDSA(1), // secp256k1
     BLS(2),   // BLS12-381
-    EDDSA(3); // ED25519
+    EdDSA(3); // ED25519
```

Comment:
The `Ed` stands for Edwards (one word) where as EC stands for Elliptic Curve (two words).  

Bouncy Castle also uses `Ed` as the start of all related classes.